### PR TITLE
Adding more log tracer for debugging and bug Fix

### PR DESCRIPTION
**Pull request by SophieGuo410**

### CODE REVIEW

File: ambry-replication/src/main/java/com/github/ambry/replication/ReplicaThread.java [hunk 1]
```diff
@@ -660,6 +660,7 @@ ReplicaMetadataResponse getReplicaMetadataResponse(List<RemoteReplicaInfo> repli
       logger.trace("Remote node: {} Thread name: {} Remote replica: {} Key from remote: {}", remoteNode, threadName,
           remoteReplicaInfo.getReplicaId(), storeKey);
       StoreKey convertedKey = storeKeyConverter.getConverted(storeKey);
+      logger.trace("SkipPredicate: {}", skipPredicate);
```
Comment (jsjtzyy) [inline ambry-replication/src/main/java/com/github/ambry/replication/ReplicaThread.java:663]: Do we have a `toString()` for `skipPredicate`?

File: ambry-api/src/main/java/com/github/ambry/replication/ReplicationSkipPredicate.java [hunk 2]
```diff
@@ -62,15 +62,18 @@ public boolean test(MessageInfo messageInfo) {
...
       if (status == Container.ContainerStatus.DELETE_IN_PROGRESS || status == Container.ContainerStatus.INACTIVE) {
-        logger.info("Container {} will be skipped during replication", container);
+        logger.trace("Container {} will be skipped during replication", container);
         return true;
       } else {
+        logger.trace("Container {} is Active", container);
         return false;
       }
     } else {
+      logger.trace("Current accountService : {}", accountService);
```
Comment (jsjtzyy) [inline ambry-api/src/main/java/com/github/ambry/replication/ReplicationSkipPredicate.java:76]: How about changing all trace to `logger.debug`?

File: ambry-replication/src/main/java/com/github/ambry/replication/ReplicaThread.java (hunk 1 above)
Comment (SophieGuo410) [reply ambry-replication/src/main/java/com/github/ambry/replication/ReplicaThread.java]: Updated.

File: ambry-api/src/main/java/com/github/ambry/replication/ReplicationSkipPredicate.java (hunk 2 above)
Comment (SophieGuo410) [reply ambry-api/src/main/java/com/github/ambry/replication/ReplicationSkipPredicate.java]: Updated.

**Pull request merged**